### ISSUE TITLE
Bindings introduced by cljs.core.match/match not understood as bindings

### ISSUE THREAD

**Issue by wolverian** [cursive-ide/cursive#1683]
[This match](https://github.com/Opetushallitus/ataru/blob/3a8aa3e1aea5e4a32e97706d0f0ca779467612d1/src/cljs/ataru/virkailija/views/virkailija_readonly.cljs#L127-L136) introduces, in its cases, new bindings `children` and `options`. Cursive says `children cannot be resolved` on them.

<img width="1259" alt="screen shot 2017-03-13 at 12 17 43" src="https://cloud.githubusercontent.com/assets/42124/23850206/14cce3cc-07e7-11e7-8259-ac535360328a.png">

Cursive should understand that these are new bindings, not names pointing to already existing bindings.

**Comment by cursive-ide** [comment]
Yes, I've been meaning to add core.match support, but have not done so yet.

**Comment by DrBuchkov** [comment]
@cursive-ide Is adding analysis support for core.match in the scope?
